README.md

```diff
@@ -27,8 +27,8 @@ up to you to manage.
    services:
      backend:
        ports:
-         - HOST_PORT:8080
+         - "8080:8080"
```

Comment:
the intent was that you define your own host port here, not copy and paste it literally, but it could be expressed better

Reply:
Idk, for me it thrown error

Reply:
probably because you copied and pasted it literally, which you aren't meant to do.

Reply:
okay, I understand